src/components/clock.tsx

```diff
@@ -0,0 +1,54 @@
+import { Component } from 'react';
+
+type Props = {
+  clockName: string;
+};
+
+type State = {
+  today: Date;
+};
+
+export class Clock extends Component<Props, State> {
+  state = {
+    today: new Date(),
+  };
+
+  timerdId = 0;
+
+  componentDidMount() {
+    this.timerdId = window.setInterval(() => {
+      this.setState({ today: new Date() });
+    }, 1000);
+  }
+
+  componentDidUpdate(_: never, prevState: State) {
+    const { today } = this.state;
+
+    if (today !== prevState.today) {
+      window.console.info(today.toUTCString().slice(-12, -4));
+    }
+  }
+
+  componentWillUnmount() {
+    window.clearInterval(this.timerdId);
+  }
+
+  render() {
+    const { today } = this.state;
+    const { clockName } = this.props;
+
+    return (
+      <div className="Clock">
+        <strong className="Clock__name">
+          {clockName}
+        </strong>
+
+        {' time is '}
+
+        <span className="Clock__time">
+          {today.toUTCString().slice(-12, -4)}
```

Comment:
You have used `.toUTCString().slice(-12, -4)` more then one time, so you can create an separate function for that.